### ISSUE TITLE
[Bug] Web: Wrong toast message when downloading a file

### ISSUE THREAD

**Issue by tprudentova** [linagora/twake-on-matrix#1387]
MacOS, Safari 

1. Open an image 
2. Click Download 
3. I see a "File saved to Downloads" message, then the system pop-up for selecting a folder appears

https://github.com/linagora/twake-on-matrix/assets/97684890/7bae4351-ed69-443d-b971-0366b80c91d5

**Comment by hoangdat** [comment]
- change to `https://pub.dev/packages/file_selector`

**Comment by Te-Z** [comment]
Same thing with file_selector. 

Looks like it is a web restriction for security reasons:

Although APIs like the File System Access API allow web applications to read from or save changes directly to files and folders on the user's device with the user's consent, this interaction is strictly regulated and does not permit retrieving the name of the folder where a file was saved after downloading. Modern browsers impose restrictions on accessing file:// URLs for security reasons, blocking navigation to these URLs from non-file:// URLs and preventing the embedding of sub-resources (such as images or scripts) from file:// URLs in pages served via HTTPS, for security and privacy reasons​​. More informations: https://textslashplain.com/2019/10/09/navigating-to-file-urls/

TLDR: on web the app can't access the name of the folder chosen to save a downloaded file. It will be saved in the default download folder (depending on OS and navigator). And this is for security policy matters.

@hoangdat shouldn't we remove the toast if the user uses the web app ?

**Comment by hoangdat** [comment]
> shouldn't we remove the toast if the user uses the web app ?

- [ ] Yes, remove this toast. 
- [ ] try to don't block user to wait download process

**Comment by hoangdat** [comment]
- [ ] web UI
- [ ] mobile UI
